tests/e2e/specs/miscellaneous/WorkpaseUsingParent.spec.ts

```diff
@@ -0,0 +1,99 @@
+/** *******************************************************************
+ * copyright (c) 2023 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { e2eContainer } from '../../configs/inversify.config';
+import { ShellExecutor } from '../../utils/ShellExecutor';
+import { InputBox, QuickOpenBox, QuickPickItem, SideBarView, ViewItem, ViewSection, Workbench } from 'monaco-page-objects';
+import { CLASSES } from '../../configs/inversify.types';
+import { ProjectAndFileTests } from '../../tests-library/ProjectAndFileTests';
+import { LoginTests } from '../../tests-library/LoginTests';
+import { Dashboard } from '../../pageobjects/dashboard/Dashboard';
+import { BrowserTabsUtil } from '../../utils/BrowserTabsUtil';
+import { BASE_TEST_CONSTANTS } from '../../constants/BASE_TEST_CONSTANTS';
+import { assert, expect } from 'chai';
+import { API_TEST_CONSTANTS } from '../../constants/API_TEST_CONSTANTS';
+import { ShellString } from 'shelljs';
+import { WorkspaceHandlingTests } from '../../tests-library/WorkspaceHandlingTests';
+import { registerRunningWorkspace } from '../MochaHooks';
+const projectAndFileTests: ProjectAndFileTests = e2eContainer.get(CLASSES.ProjectAndFileTests);
+const loginTests: LoginTests = e2eContainer.get(CLASSES.LoginTests);
+const browserTabsUtil: BrowserTabsUtil = e2eContainer.get(CLASSES.BrowserTabsUtil);
+const dashboard: Dashboard = e2eContainer.get(CLASSES.Dashboard);
+const shellExecutor: ShellExecutor = e2eContainer.get(CLASSES.ShellExecutor);
+const workspaceHandlingTests: WorkspaceHandlingTests = e2eContainer.get(CLASSES.WorkspaceHandlingTests);
+function executeArbitraryShellScript(command: string): string {
+	const output: ShellString = shellExecutor.executeCommand(command);
+	if (output.stderr.length > 0) {
+		assert.fail(output.stderr);
+	}
+	return output.stdout;
+}
+suite('Workspace using a parent test suite', function (): void {
+	loginTests.loginIntoChe();
+	let podName: string = '';
+
+	test('Create a workspace using a parent', async function (): Promise<void> {
+		const factoryUrl: string = `${BASE_TEST_CONSTANTS.TS_SELENIUM_BASE_URL}/dashboard/#https://github.com/testsfactory/parentDevfile`;
+		await dashboard.waitPage();
+		await browserTabsUtil.navigateTo(factoryUrl);
+		await workspaceHandlingTests.obtainWorkspaceNameFromStartingPage();
+		registerRunningWorkspace(WorkspaceHandlingTests.getWorkspaceName());
+		await projectAndFileTests.waitWorkspaceReadinessForCheCodeEditor();
+		// sometimes the trust dialog does not appear at first time, for avoiding this problem we send click event for activating
+		await new Workbench().click();
```

Comment:
could it be inside projectAndFileTests.performTrustAuthorDialog() method?

Reply:
implemented: https://github.com/eclipse/che/pull/22544